test/resources/IssuerFraudRecords.spec.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var stripe = require('../testUtils').getSpyableStripe();
+var expect = require('chai').expect;
+
+describe('IssuerFraudRecord Resource', function() {
+  describe('retrieve', function() {
+    it('Sends the correct request for issfr ID', function() {
+      stripe.issuerFraudRecords.retrieve('issfr_123');
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'GET',
+        url: '/v1/issuer_fraud_records/issfr_123',
+        data: {},
+        headers: {},
+      });
+    });
+    it('Sends the correct request for charge ID', function() {
+      stripe.issuerFraudRecords.listFromCharge('ch_123456789');
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'GET',
+        url: '/v1/issuer_fraud_records?charge=ch_123456789',
+        data: {},
+        headers: {},
+      });
+    });
```

Comment:
I think we can safely drop this test case, but if we don't, it should go in the `list` block below instead of this `retrieve` block.